### PR TITLE
Soumya

### PR DESCRIPTION
**Pull request by hellosibun**

### CODE REVIEW

File: FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import com.upgrad.FoodOrderingApp.service.businness.CategoryServices;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/category")
+public class CategoryController {
+    @Autowired
+    CategoryServices
```
Comment (ashwini008) [inline FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/CategoryController.java]: should be private CategoryService categoryService

File: FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/OrderController.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    @CrossOrigin
+    @RequestMapping(method = RequestMethod.GET, path="/coupon/{coupon_name}")
+    public ResponseEntity<CouponDetailsResponse> getCouponByCouponName(@RequestHeader(value = "authorization") final String authorization, @PathVariable(value = "coupon_name") final String couponName) throws AuthorizationFailedException, CouponNotFoundException {
+
+        //get access-token from the request header
+        String accessToken = authorization.split("Bearer ")[1];
+
+        //Calls getCouponByCouponName of orderService to get the coupon by name from DB
+        CouponEntity couponEntity =
```
Comment (ashwini008) [inline FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/OrderController.java:21]: We will have issues when we pull this. Fix this.

File: FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/CategoryController.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.upgrad.FoodOrderingApp.api.controller;
+
+import com.upgrad.FoodOrderingApp.service.businness.CategoryServices;
```
Comment (ashwini008) [inline FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/CategoryController.java]: Incorrect import
File name is CategoryService

File: FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/CategoryController.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+import java.util.UUID;
+
+// Category Controller Handles all  the Category related endpoints
+
+@RestController
+@RequestMapping("/category")
+public class CategoryController {
+
+    @Autowired
+    CategoryService categoryService;
```
Comment (ashwini008) [inline FoodOrderingApp-api/src/main/java/com/upgrad/FoodOrderingApp/api/controller/CategoryController.java:30]: set as private

**Review by hellosibun — COMMENTED** [review]
xml and iml files removed

**Pull request merged**